model/scratch/dffml-model-anomalydetection/tests/test1.csv

```diff
@@ -0,0 +1,92 @@
+A,B,C,D,E,F,G,H,I,J,K,Y
```

Comment:
Let's have the test and train data generated by some Python function. Let cut it down to 2 features and then Y.